Asterisk.2013/Asterisk.NET/Manager/Event/AgentRingNoAnswerEvent.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace AsterNET.Manager.Event
+{
+    /// <summary>
+    ///     Raised when a queue member is notified of a caller in the queue and fails to answer.
+    ///     
```

Comment:
I don't think this works as a linebreak. You have to use `<br/>` instead.

Reply:
more for readability in comments, i can remove the extra whitespace if wanted?